frontend/public/components/search.tsx

```diff
@@ -183,17 +183,26 @@ const SearchPage_: React.FC<SearchProps> = (props) => {
     if (!model) {
       return '';
     }
-    const { labelPlural, apiVersion, apiGroup } = model;
+    const { labelPlural, labelPluralKey, apiVersion, apiGroup } = model;
     return (
       <span className="co-search-group__accordion-label">
-        {labelPlural}{' '}
+        {labelPluralKey ? t(labelPluralKey) : labelPlural}{' '}
         <span className="text-muted show small">
           {apiGroup || 'core'}/{apiVersion}
         </span>
       </span>
     );
   };
 
+  const getChipText = (item: string) => {
+    const model = modelFor(item);
+    // API discovery happens asynchronously. Avoid runtime errors if the model hasn't loaded.
+    if (!model || !model.labelKey) {
+      return kindForReference(item);
```

Comment:
I would fall back to `model.label` since we have a null check for the model above. It should always be defined.